src/Umbraco.Core/Persistence/PetaPocoSqlExtensions.cs

```diff
@@ -15,6 +14,57 @@ namespace Umbraco.Core.Persistence
     /// </summary>
     public static class PetaPocoSqlExtensions
     {
+        /// <summary>
+        /// Defines the column to select in the generated SQL query
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="sql">Sql object</param>
+        /// <param name="fields">Columns to select</param>
+        /// <returns></returns>
+        [Obsolete("Use the overload specifying ISqlSyntaxProvider instead")]

```

Comment:
I see why you did this, but instead, let's just fix the source of the problem. I'd add a field on the `BaseUsingSqlCeSyntax` class that just initiliazes a `new SqlServerSyntaxProvider()` which you can then use in your tests. It's not very nice to introduce a new public method that is immediately obsolete. ;-)

Reply:
Ok, I'll have a look at that :-D

Reply:
Should be `SqlCeSyntaxProvider` of course since the tests are using CE. :-)